translations/en.json

```diff
@@ -658,7 +658,7 @@
     "invited": "invited",
     "invite-button": "Invite",
     "invite-receive-account": "Account to receive your referral bonus",
-    "invite-instruction": "How it works",
+    "how-it-works": "How it works",
```

Comment:
what about other languages ?

Reply:
Will sync with Jinho.

Reply:
i mean if you change a key you have to change it for every language , or we have it only in en ?

Reply:
No, we have it everywhere. Will change it manually.